package.json

```diff
@@ -1,43 +1,55 @@
 {
   "name": "cbioportal-frontend",
+  "private": true,
+  "version": "0.0.0",
+  "workspaces": {
+    "packages": [".", "packages/*"]
+  },
   "description": "",
   "scripts": {
-    "start": "./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cross-env NODE_ENV=development webpack-dev-server",
-    "startSSL": "./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cross-env NODE_ENV=development webpack-dev-server --https",
+    "start": "lerna run watch --parallel",
+    "startSSL": "lerna run watchSSL --parallel",
+    "watch": "./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cross-env NODE_ENV=development webpack-dev-server",
+    "watchSSL": "./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cross-env NODE_ENV=development webpack-dev-server --https",
     "clean": "rimraf dist tsDist common-dist",
-    "build": "yarn run clean && yarn run compileOqlParser && yarn run buildDLL:prod && cross-env NODE_ENV=production NODE_OPTIONS=--max-old-space-size=2048 webpack",
+    "build": "./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && yarn run buildAll",
+    "buildAll": "yarn run buildModules && yarn run buildMain",
```

Comment:
currently, jitpack is the only use case where we don't need to set env vars, so we call `buildAll` instead of `build` there.